patternlab/styleguide/source/_patterns/01-atoms/09-media/site-logo.twig

```diff
@@ -1,5 +1,5 @@
 <div class="ma__site-logo">
-  <a href="/" title="Mass.gov home page">
+  <a href="/" id="ma__site-logo-link" title="Mass.gov home page">

```

Comment:
What is this id used for?

Reply:
This was added so I could catch the focus to close the menu. The rest of the code for it was waiting for the utility menu. That code has now been added to use this link ID

Reply:
Sounds good.